LiskHQ/lisk-desktop
# Handle page breakdown and resolve password index positioning bug

### What was the problem?

This PR resolves #5027, #5026

### How was it solved?

- [x] Ignore tabIndex on the input type toggle button
- [x] Fix Issue on wallet page crashing

### How to test

#### How to test solution of #5027
- Navigate to the add account by secret passphrase.
- Go through the flow until the point of entering a password and account name
- You should be able to navigate the input field smoothly by clicking on the tab key.

#### How to test solution of #5026
- Change the url in the useTokenBalance hook to a wrong one (this would allow you mock a failed token balance api call)
- Navigate to the wallet page.
- You should now be seeing an error section with a retry button instead of the application crashing.
